source/MRCuda/MRCudaPolylineProjection.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include "exports.h"
+#include "MRMesh/MRVector2.h"
+#include "MRMesh/MRPolyline.h"
+#include "MRMesh/MRDistanceMap.h"
+
+namespace MR { namespace Cuda
+{
+
+MRCUDA_API DistanceMap distanceMapFromContours( const MR::Polyline2& polyline, const ContourToDistanceMapParams& params );
```

Comment:
Please add comment about this function, and mb rename file to MRCudaContoursDistanceMap